templates/gerrit/replication.config

```diff
@@ -0,0 +1,35 @@
+[gerrit]
+  autoReload = true
+  defaultForceUpdate = true
+[remote "git"]
+  url = <%= node['git-daemon']['user'] %>@<%= node['git']['hostname'] %>:<%= node['git-daemon']['home'] %>/repositories/${name}.git
+  authGroup = Git Mirror
+  mirror = true
+  threads = 8
+  timeout = 120
+[remote "forge"]
+  url = git@forge.typo3.org:repositories/${name}.git
+  push = +refs/heads/*:refs/heads/*
+  push = +refs/tags/*:refs/tags/*
+  authGroup = Forge Mirror
+  mirror = true
+  threads = 8
+  timeout = 120
+[remote "github/TYPO3/TYPO3.CMS"]
+  projects = Packages/TYPO3.CMS
+  url = https://github.com/TYPO3/TYPO3.CMS.git
+  push = +refs/heads/*:refs/heads/*
+  push = +refs/tags/*:refs/tags/*
+  authGroup = Git Mirror
+  mirror = true
+  timeout = 120
```

Comment:
Is `threads = 8` missing here by accident or on purpose?

Reply:
Was indeed by accident (although copied&pasted from the old config).

Probably, the thread count doesn't matter for our tiny repos. But could be fixed.